template-for-proof/Makefile

```diff
@@ -1,8 +1,15 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-HARNESS_ENTRY=harness
-HARNESS_FILE=<__FUNCTION_NAME__>_harness
+# Additional flags for CBMC
+CBMCFLAGS +=
+CHECKFLAGS +=
+
+# Disabled CBMC flags
+DISABLED_FLAGS =
```

Comment:
I don't think these three definitions (duplicates from the makefile.common) are needed here.  It might be prudent to add some documentation here explaining the use of DISABLED_FLAGS.  But if your code works, it works even when DISABLED_FLAGS is undefined.